doc/psud/PSU_daemon_design.md

```diff
@@ -24,6 +24,17 @@ The purpose of PSU daemon is to collect platform PSU data and trigger proper act
   - whether the PSU voltage exceeds the minimal and maximum thresholds
   - whether the PSU temperature exceeds the threshold
   - whether the total PSU power consumption exceeds the budget (modular switch only)
+  - whether PSU power consumption exceeds the PSU threshold
+
+### 1.1 PSU power threshold check
+
+#### 1.1.1 Why we need it
+
+An ethernet switch is typically equipped with more than one PSUs for redundancy. It can be deployed in different scenarios with different kinds of xSFP modules, traffic type and traffic load. All these factors affect the power consumption of an ethernet switch.
+
+On some platforms, the capacity of a single PSU is not large enough to afford all the components and xSFP modules running at the highest performance at the same time. In this case, we do not have redundancy any longer. Users should be notified of that in order to take steps in advance, which is achieved via periodically checking the current power of PSUs against their maximum allowed power, AKA, power thresholds.
```

Comment:
I think the user should only know if redundancy is avaialble or not. 
Are we expecting any manual action for just an FYI?

Reply:
I think it's only an FYI. Will remove the term `in order to take steps in advance`.